CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
+  - Remove start_date attribute
+  - Rename stop_date attribute to end
+- In formulas:
+  - Remove SimpleFormula and DatedFormula. Any Formula acts like old DatedFormula
+  - Remove @dated_function: start goes to formula name suffix, stop is deduced from other formulas start dates and variable end attribute 
+
+#### New features
+
+- Introduce `formula_YYYY[_MM[_DD]]`
+  - Allows for formula start instant definition
+- Introduce `end = 'YYYY-MM-DD'`
+  - Allows for variable end date definition in string format
+- Change ETERNITY period effect
+  - Allows for dated Variable definition on ETERNITY period
```

Comment:
Dated variables don't exist anymore, so mentioning them in the changelog is not great.

Suggestion : 

- Remove restriction that prevented to have formula changes over time for a variable with `ETERNITY` as a definition period.